### PR TITLE
Improve README and LICENSE

### PR DESCRIPTION
**Pull request by ben221199**
This PR makes some improvements to both the README and the LICENSE file.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+The MIT License (MIT)
```
Comment (RobbinBaauw) [inline LICENSE.md:1]: This change is not really necessary: the `LICENSE` file has no extension by default

File: README.md [hunk 2]
```diff
@@ -1,25 +1,50 @@
...
+(Or use provided IntelliJ runconfigs.)
+
+For the client:
+`yarn run serve`
+
+## Feature Requests / Bug Reports
+
+Create a GitHub issue on this repository, and we will try to take a look at it.
+
+## How does it work?
```
Comment (RobbinBaauw) [inline README.md:39]: This explanation doesn't really say anything and can be removed all together

**Review by RobbinBaauw — CHANGES_REQUESTED** [review]

File: LICENSE.md (hunk 1 above)
Comment (ben221199) [reply LICENSE.md:1]: https://github.com/IQAndreas/markdown-licenses

Comment (NULLx76) [reply LICENSE.md:1]: The MIT license doesn't have any formatting anyways so I think its quite pointless

File: README.md (hunk 2 above)
Comment (ben221199) [reply README.md:39]: This test was already in the old README file, so I kept it. Now I removed it in commit 64b42f1862afa021427727893abcbfd4b02b40e6.

File: LICENSE.md (hunk 1 above)
Comment (ben221199) [reply LICENSE.md:1]: The title has heading formatting.

**Pull request closed without merging**